styles/style.css

```diff
@@ -53,6 +53,11 @@ ul li {
   display: flex;
   align-items: center;
   cursor: pointer;
+  transition: transform 1s;
```

Comment:
Looks good 🤝, But you can try to add rotate 360deg it will give you a good experience.